src/core/system_titles.cpp

```diff
@@ -0,0 +1,1124 @@
+// Copyright 2023 Citra Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include <vector>
+#include "core/hle/service/am/am.h"
+#include "core/hle/service/fs/archive.h"
+#include "core/system_titles.h"
+
+namespace Core {
+
+struct SystemTitle {
+    std::string name;
```

Comment:
```suggestion
    std::string_view name;
```

Would allow usages of this struct to be constexpr, which is fine since all usages seem to be using string literals

Reply:
Even with this change, ```system_titles``` still can't be constexpr with the same message as before.

Reply:
Ah, yeah, since it has a vector as one of its members. Nothing that can be done about that other than turning it into a std::array and marking the end of valid entries with something that signifies an invalid entry, which can be annoying, so don't worry about that I guess